--- gdal-3.5.3/configure	2022-09-03 00:20:11
+++ gdal-3.5.3-fix/configure	2022-11-13 21:27:54
@@ -37142,7 +37142,7 @@
 
       HAVE_GEOS="no"
 
-      GEOS_LIBS="`${GEOS_CONFIG} --ldflags` -lgeos_c"
+      GEOS_LIBS="`${GEOS_CONFIG} --static-clibs` -lstdc++"
       GEOS_CFLAGS="`${GEOS_CONFIG} --cflags`"
       GEOS_VERSION="`${GEOS_CONFIG} --version`"

--- gdal-3.5.3/frmts/mrf/LERCV1/Lerc1Image.orig.cpp	2022-10-21 16:57:17
+++ gdal-3.5.3/frmts/mrf/LERCV1/Lerc1Image.cpp	2024-04-21 12:11:54
@@ -22,6 +22,7 @@
 */
 
 #include "Lerc1Image.h"
+#include <stdint.h>
 #include <cmath>
 #include <cfloat>
 #include <string>

--- gdal-3.5.3/gcore/gdaljp2metadatagenerator.orig.cpp	2024-04-23 00:41:21
+++ gdal-3.5.3/gcore/gdaljp2metadatagenerator.cpp	2024-04-23 00:42:58
@@ -355,7 +355,12 @@
 /************************************************************************/
 
 static void GDALGMLJP2XPathErrorHandler( void * /* userData */,
-                                         xmlErrorPtr error)
+#if LIBXML_VERSION >= 21200
+                                        const xmlError *error
+#else
+                                        xmlErrorPtr error
+#endif
+)
 {
     if( error->domain == XML_FROM_XPATH &&
         error->str1 != nullptr &&
